Doc/reference/expressions.rst

```diff
@@ -1874,10 +1878,12 @@ Expression lists
    single: , (comma); expression list
 
 .. productionlist:: python-grammar
-   expression_list: `expression` ("," `expression`)* [","]
-   starred_list: `starred_item` ("," `starred_item`)* [","]
-   starred_expression: `expression` | (`starred_item` ",")* [`starred_item`]
-   starred_item: `assignment_expression` | "*" `or_expr`
+   starred_expression: ["*"] `or_expr`
```

Comment:
`or_expr` is valid for what comes after `*`, but starred_expression now also gets used (via expression_list) in the rule for list_display, and there other expressions are valid, e.g. comparisons. I think this should be:
```suggestion
   starred_expression: "*" `or_expr` | `expression`
```

Reply:
I think `expression` is already included in `expression_list` through `assignment_expression` - though maybe it would be worth adding the extra `| expression` here to make it clear?

Reply:
Ah you're right, I missed that `assignment_expression` includes `expression`.

But I think there's still something wrong here: `yield a == b, c == d` is valid, but `a == b` is a `comparison`, and the grammar for `yield_list` indicates that after the comma we can only have a `starred_expression`.